### PR TITLE
blog: November newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Last-second-save, as is traditional ;)

It's already dated for tomorrow, since I expect that'll be accurate. I need to add the next demo (15th Dec) but I haven't got the URL yet - will do that later tonight.

Seems light compared to recent months, what did I miss? @ehelms should I mention the moving of the Katello website?

### CODE REVIEW

**Comment by ehelms** [comment]
Please do.

**Comment by GregSutcliffe** [comment]
@ehelms done. Any other additions anyone?

**Review by ohadlevy — APPROVED** [review]

File: _posts/2016-11-30-foreman-community-newsletter---november-2016.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#### DevConf.cz / FOSDEM / CfgMgmtCamp Ghent - February 2017
+
+The CFPs for DevConf, FOSDEM, and CfgMgmtCamp are now closed, and the responses
+look great - thanks to all who submitted talks! The schedules will be published
+in due course, once the talks are selected and the speakers confirmed, but you
+already know it'll be great. Start booking your travel!
+
+We're also happy to confirm that the Foreman hack day will happen again this
+year, on the 8th of February. I'll be looking to gather opinion on what format
+people want shortly, so keep an eye on the [mailinglist][users_list] for that!
```
Comment (mmoll) [inline _posts/2016-11-30-foreman-community-newsletter---november-2016.md:110]: `userslist`

Comment (dLobatog) [reply _posts/2016-11-30-foreman-community-newsletter---november-2016.md:110]: I'll fix that on merge 😄

**Comment by dLobatog** [comment]
Merged as 0f5458680ab9d68cfd60a509cb22608e151ca4ea - thanks @GregSutcliffe @ohadlevy @ehelms @mmoll

**Pull request closed without merging**